### PR TITLE
Remove radium from NewListItem

### PR DESCRIPTION
**Pull request by megcrenshaw**
I started working on this last week, so wanted to finish it up! This continues the work of deprecating radium. There's an eyes test for this, so I ran it with drone to ensure it worked. @fisher-alice I'm requesting your review because I know you did work on this component recently!

Locally:
Screen Shot 2022-08-18 at 10 42 08 AM
Screen Shot 2022-08-18 at 10 42 15 AM

Compared to prod:
Screen Shot 2022-08-18 at 10 41 49 AM
Screen Shot 2022-08-18 at 10 41 40 AM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by fisher-alice — COMMENTED** [review]
The updated code with scss looks so clean - nice work!
Could you please include a screenshot of what the new costume button looks like compared to the two buttons below it while the bee animation is selected?
Mark asked that the new costume button have the same width as the other buttons when they're not selected. See below as reference. Thanks!

Screen Shot 2022-08-19 at 9 04 04 AM

File: apps/src/p5lab/AnimationTab/new-list-item.module.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    cursor: pointer;
+    box-shadow: none;
+
+    .border {
+      background-color: $polar-blue;
+      border: solid 4px $polar-blue;
+    }
+
+    .addButton {
+      color: $white
```
Comment (bencodeorg) [inline apps/src/p5lab/AnimationTab/new-list-item.module.scss]: Nit: semicolon. Also, do we like newlines at end of these files?

File: apps/src/p5lab/AnimationTab/NewListItem.jsx [hunk 2]
```diff
@@ -15,90 +15,22 @@ class NewListItem extends React.Component {
...
-                styles.animationName,
-                hovered && styles.borderHovered_animationName
-              ]}
-            >
-              {this.props.label}
-            </div>
+        <div className={style.wrapper}>
+          <div className={style.border}>
+            <i className={classNames(style.addButton, 'fa fa-plus-circle')} />
+            <div className={style.animationName}>{this.props.label}</div>
```
Comment (bencodeorg) [inline apps/src/p5lab/AnimationTab/NewListItem.jsx:28]: Is the animation-name class name no longer needed?

File: apps/src/p5lab/AnimationTab/new-list-item.module.scss (hunk 1 above)
Comment (megcrenshaw) [reply apps/src/p5lab/AnimationTab/new-list-item.module.scss]: Good catch! It looks like our linter should catch that https://github.com/stylelint/stylelint-config-standard/blob/main/index.js#L66 –– it looks like the prettier config conflicts. I took out the prettier file and fixed the file with the linter, which fixes the semicolon! Will bring this to the larger group.

As for newlines, it looks like the linter complains about >1 new line, but 0 or 1 newlines both pass (after blocks as well). I've added it for consistency, but that might be another linter candidate?

File: apps/src/p5lab/AnimationTab/NewListItem.jsx (hunk 2 above)
Comment (megcrenshaw) [reply apps/src/p5lab/AnimationTab/NewListItem.jsx:28]: Good call. I couldn't find that className in our repo (it's used as an attribute in some places https://github.com/code-dot-org/code-dot-org/blob/staging/apps/style/common.scss#L47)

It looks like some of the other classNames were used as identifiers in eyes tests, so I'm wondering if that was used at some point here as well.

**Comment by megcrenshaw** [comment]
@fisher-alice Good call! Didn't even know this had a different label in Spritelab.

Without the focus on the bee:
image

With the focus on the bee:
image

Does that seem right?

File: apps/src/p5lab/AnimationTab/NewListItem.jsx (hunk 2 above)
Comment (bencodeorg) [reply apps/src/p5lab/AnimationTab/NewListItem.jsx:28]: TIL about `animation-name` as a CSS property :)

**Review by bencodeorg — APPROVED** [review]
Another one bites the dust! ✅

File: apps/src/p5lab/AnimationTab/NewListItem.jsx (hunk 2 above)
Comment (megcrenshaw) [reply apps/src/p5lab/AnimationTab/NewListItem.jsx:28]: I also didn't know about that until this work!

**Comment by fisher-alice** [comment]
> @fisher-alice Good call! Didn't even know this had a different label in Spritelab.
> 
> Without the focus on the bee: image
> 
> With the focus on the bee: image
> 
> Does that seem right?

Yes - looks good, thanks!

**Pull request merged**